### PR TITLE
Translation of post titles on Feed

### PR DESCRIPTION
**Pull request by YuriyLu**

### CODE REVIEW

File: app/containers/Questions/Content/Body/LanguageLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import { BG_PRIMARY } from 'style-constants';
+
+import Container from 'components/Labels/QuestionType';
+
+import Popover from './Popover';
+import { Languages } from 'icons/index';
+import { languagesWithDescriptions } from 'app/i18n';
+
+const LabelItem = styled.span`
```
Comment (Lampardks) [inline app/containers/Questions/Content/Body/LanguageLabel.tsx:13]: use emotion css

File: app/containers/Questions/Content/Body/LanguageLabel.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  > div {
+    position: inherit;
+  }
+`;
+
+const LanguageLabel: React.FC<{ isSearch: boolean; language: string }> = ({
```
Comment (Lampardks) [inline app/containers/Questions/Content/Body/LanguageLabel.tsx:19]:
```
const LanguageLabel: React.FC<{ isSearch?: boolean; language: string }> = ({
```

File: app/containers/Questions/Content/Body/LanguageLabel.tsx (hunk 1 above)
Comment (YuriyLu) [reply app/containers/Questions/Content/Body/LanguageLabel.tsx]: done

File: app/containers/Questions/Content/Body/LanguageLabel.tsx (hunk 2 above)
Comment (Lampardks) [reply app/containers/Questions/Content/Body/LanguageLabel.tsx]: not fixed

**Review by Lampardks — APPROVED** [review]

**Pull request merged**